includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -708,7 +707,11 @@ private function get_courses( $args ) {
 
 		add_filter( 'posts_clauses', [ $this, 'filter_courses_by_last_activity' ] );
 		add_filter( 'posts_clauses', [ $this, 'add_days_to_completion_to_courses_queries' ] );
+		if ( isset( $args['orderby'] ) && ( 'count_of_completions' === $args['orderby'] ) ) {
```

Comment:
I think `days_to_completion` should also work here?

Reply:
So **days_to_completion** is actually not working. 

The issue is that the value for **days_to_completion** we get from the query is not the one that we display in the table. In the table (and this is true for both lessons and courses) we show the value that is divided by **lesson_completed_count**.